### PR TITLE
Clean up messages and docs 

### PR DESCRIPTION
**Pull request by LiNk-NY**
Based on recommendations in #144

### CODE REVIEW

File: R/repositories.R [hunk 1]
```diff
@@ -21,18 +29,19 @@ BINARY_BASE_URL <- "https://bioconductor.org/packages/%s/container-binaries/%s"
...
         txt <- paste(
             "'getOption(\"repos\")' replaces Bioconductor standard ",
-            "repositories, see '?repositories' for details"
+            "repositories, see ",
+            "'help(\"repositories\", package = \"BiocManager\")' for details."
         )
         fmt <- paste0(
             .msg(txt, exdent = 0),
-            "\n\nreplacement repositories:",
+            "\n\nReplacement repositories:",
```
Comment (mtmorgan) [inline R/repositories.R:37]: I find the extra newline here distracting

**Review by mtmorgan — APPROVED** [review]

File: R/repositories.R (hunk 1 above)
Comment (LiNk-NY) [reply R/repositories.R:37]: @mtmorgan The difference is the capital `R` in 'replacement'

Comment (mtmorgan) [reply R/repositories.R:37]: I understand, but since you were revising I thought I'd mention...

Comment (LiNk-NY) [reply R/repositories.R:37]: Ah, did you mean the extra `\n`? Yes, I can remove it.

**Pull request merged**